htmlhint/index.d.ts

```diff
@@ -23,7 +23,7 @@ export interface Rule {
 }
 
 export interface RuleSet {
-    [id: string]: boolean;
+    [id: string]: boolean | string;
```

Comment:
Also, I think `type` namings should be Pascal case similar to Interfaces
ex: `type WhiteSpaceRule = "tab" | "space" | "mixed" | "disabled";`